src/main/java/com/cleanroommc/groovyscript/compat/mods/ModSupport.java

```diff
@@ -28,6 +29,7 @@ public class ModSupport implements IGroovyPropertyGetter {
 
     public static final Container<EnderIO> ENDER_IO = new Container<>("enderio", "Ender IO", EnderIO::new, "eio");
     public static final Container<JustEnoughItems> JEI = new Container<>("jei", "Just Enough Items", JustEnoughItems::new, "hei");
+    public static final Container<Thaumcraft> THAUMCRAFT = new Container<>("thaumcraft", "Thaumcraft", Thaumcraft::new);
```

Comment:
may add `tc` here

Reply:
added in commit: [https://github.com/CleanroomMC/GroovyScript/commit/4a9593468ed6525cba83c3803cf1958118c8282d](https://github.com/CleanroomMC/GroovyScript/commit/4a9593468ed6525cba83c3803cf1958118c8282d)